### PR TITLE
GUMGUM Bid Adapter: Add GPID for TTD 

### PR DESCRIPTION
**Pull request by estavillo**
## Type of change

- [x] Feature

## Description of change
Add gpid from bid request

- [x] official adapter submission

### CODE REVIEW

**Comment by estavillo** [comment]
Not sure why those unit tests are failing, I did not edit ParrableId's code. :(

**Comment by ChrisHuie** [comment]
@estavillo Kicked off the tests again. There are a few flaky tests we are currently trying to unwind

**Comment by ChrisHuie** [comment]
@estavillo can you please add some unit test coverage for this pr please too

**Comment by lbenmore** [comment]
@ChrisHuie a unit test was added in; plz let us know if this is sufficient to proceed

**Comment by ChrisHuie** [comment]
Looks good to me

**Review by ChrisHuie — APPROVED** [review]

File: test/spec/modules/gumgumBidAdapter_spec.js [hunk 1]
```diff
@@ -189,6 +189,13 @@ describe('gumgumAdapter', function () {
       expect(bidRequest.data).to.not.have.property('irisid');
     });
 
+    it('should set the global placement id (gpid)', function () {
+      const req = { ...bidRequests[0], ortb2Imp: { ext: { data: { adserver: { name: 'test', adslot: 123456 } } } } }
+      const bidRequest = spec.buildRequests([req])[0];
+      expect(bidRequest).to.have.property('gpid');
+      expect(bidRequest.gpid).to.equal(123456);
```
Comment (patmmccann) [inline test/spec/modules/gumgumBidAdapter_spec.js:196]: this will always be a string

File: modules/gumgumBidAdapter.js [hunk 2]
```diff
@@ -318,6 +325,7 @@ function buildRequests (validBidRequests, bidderRequest) {
       sizes,
       url: BID_ENDPOINT,
       method: 'GET',
+      gpid: gpid,
```
Comment (patmmccann) [inline modules/gumgumBidAdapter.js:328]: unless you put something in data., as you have data.pi above, or data.fp, it doesn't appear to flow through to your endpoint

**Pull request merged**